### PR TITLE
Feature hgvs to genomic variation

### PR DESCRIPTION
**Pull request by nzhun**
The HGVS parser finished

### CODE REVIEW

**Comment by nzhun** [comment]
The HGVS parser finished

File: jannovar-cli/src/main/java/de/charite/compbio/jannovar/Jannovar.java [hunk 1]
```diff
@@ -133,13 +135,12 @@ private static void printTopLevelHelp() {
...
 		System.err.println("         annotate      functional annotation of VCF files");
 		System.err.println("         annotate-pos  functional annotation of genomic change");
+		System.err.println("         hgvs-to-genomic-pos  parse hgvs to genomic position");
 		System.err.println("");
 		System.err.println("Example: java -jar de.charite.compbio.jannovar.jar download -d hg19/ucsc");
 		System.err.println("         java -jar de.charite.compbio.jannovar.jar db-list");
-		System.err
-				.println("         java -jar de.charite.compbio.jannovar.jar annotate -d data/hg19_ucsc.ser -i variants.vcf");
-		System.err
-				.println("         java -jar de.charite.compbio.jannovar.jar annotate-pos -d data/hg19_ucsc.ser -c 'chr1:12345C>A'");
```
Comment (visze) [inline jannovar-cli/src/main/java/de/charite/compbio/jannovar/Jannovar.java:142]: Why you removed this line. please put it back again!

File: jannovar-cli/src/main/java/de/charite/compbio/jannovar/cmd/JannovarAnnotationCommand.java [hunk 2]
```diff
@@ -21,7 +21,7 @@
 	/** Map of Chromosomes, used in the annotation. */
 	protected ImmutableMap<Integer, Chromosome> chromosomeMap = null;
 	
-	protected JannovarData data = null;
+	protected  JannovarData data = null;
```
Comment (visze) [inline jannovar-cli/src/main/java/de/charite/compbio/jannovar/cmd/JannovarAnnotationCommand.java]: Your space is wrong. So look up the changes before are use git commit.
And do not use auto formating.

**Comment by visze** [comment]
Hey,

thanks a lot. There are some issues. here are the biggest ones:
- [ ] Javadoc comments are missing in all of your code
- [ ] use HTSJDK library to write out VCF files (see HGVSToGenomicPositionCommand
- [ ] we should introduce a new cli command: hgvs-to-vcf (HGVS file to VCFs)  and keep the old idea of hgvs-to-genomic-pos (ONE CHANGE)
- [ ] move test/test.txt into the resource test folder and write junit tests to test your code

some small things:
- [ ] please remove jannovar-cli/header.txt 
- [ ] See my comments about Jannovar.java
- [ ] See comment about JannovarAnnotationCommand.java
- [ ] we have to check if ANTLR (jannovar-hgvs) provides this class HGVS.java
- [ ] remove file mvn.sh
- [ ] class utils is not needed

**Pull request closed without merging**